ui/v2.5/src/locales/en-GB.json

```diff
@@ -498,7 +499,7 @@
   "image_count": "Image Count",
   "images": "Images",
   "images-size": "Images size",
-  "include_child_studios": "Include child studios",
```

Comment:
Please remove existing `include_child_studios` from the other localization files, so that we don't end up with vestigial entries.